.github/workflows/tests.yml

```
@@ -107,6 +107,23 @@ jobs:
           pip install -vv -e .
           SKIP_BZIP_TEST=true pytest -vv fitsio
 
+      - name: test sdist
+        shell: bash -l {0}
+        run: |
+          rm -rf build*
+          rm -rf $HOME/cfitsio-static-install
+          find . -name "*.so" -type f -delete
+          python setup.py sdist
+          pushd dist/
+          fname=$(ls fitsio*.gz)
+          tar xvfz "$fname"
+          dname=$(echo "$fname" | sed 's/\.tar\.gz//')
+          pushd $dname
+          pip install -vv -e .
+          SKIP_BZIP_TEST=true pytest -vv fitsio
+          popd
+          popd
```

```suggestion
          pip install -vv --no-deps --no-build-isolation dist/*.tar.gz
```

This version does not produce an editable install, but you can install the sdist directly without unpacking it. It might be worth check that the install itself doesn't fail this way even if we cannot run the tests.

---

How would you  run tests?

---

Are you suggesting we add another section to the CI which just runs that command but does not do tests?

---

yes

---

you can try something like this

```
          rm -rf dist/*
          python setup.py sdist
          pip install -vv --no-deps --no-build-isolation dist/*.tar.gz
          cd ..
          python -c "import conda_forge_tick; assert conda_forge_tick.__version__ != '0.0.0'"
          cd -
          pip uninstall conda-forge-tick --yes
```

---

`ModuleNotFoundError: No module named 'conda_forge_tick'`

---

yeah you need `fitsio` there. ;p